CryptoPrices/Sources/Styleguide/Sources/Styleguide/Helpers/View+Size.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  View+Size.swift
+//  Styleguide
+//
+//  Created by Minh Pham on 04/01/2023.
+//
+// swiftlint:disable file_types_order
+
+import SwiftUI
+
+public extension View {
+
+    func readSize(onChange: @escaping (CGSize) -> Void) -> some View {
```

Comment:
Feel free to ignore this: considering the method name is `readSize`, I think the filename `View+ReadSize` would be a little better one 🙏